vision_datasets/common/data_manifest.py

```diff
@@ -764,3 +774,27 @@ def create_dataset_manifest(coco_file_path_or_url, data_type, container_sas_or_r
         images.sort(key=lambda x: x.id)
 
         return DatasetManifest(images, labelmap, data_type)
+
+    def _load_image_matting_label_from_file(image_label_file_path):
+        """
+        Load image matting label from a zip file
+        Arg:
+            image_label_file_path: an annotation file path for the image. Two formats are supported:
+                1. a file path containing the image file name e.g. foo/bar.jpg
+                2. a file path containing the zip file name and the image name, connecting with '@',  e.g. '/tmp/foo.zip@bar.jpg'
+        Return:
+            A PIL image object
+        """
+        if '@' in image_label_file_path:
```

Comment:
There is a file_reader class taking care of file access, no need to repeat the logic check here